### PR TITLE
[Feature]- Adding JDBC-presto connector

### PR DESCRIPTION
**Pull request by sanjaychitransh**
…ed code PR Add presto connector

#3774

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        }
+        if (dbServer != null) {
+            dbServer.close();
+        }
+    }
+
+    @TestTemplate
+    @DisplayName("JDBC-Presto end to end test")
+    public void testJdbcPresto(TestContainer container) {
+        assertHasData();
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:104]: You need to submit a job to check connector-jdbc connect to presto

**Review by hailin0 — COMMENTED** [review]
1. update `config/plugin_config`
2. update `plugin-mapping.properties`
3. update docs

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java (hunk 1 above)
Comment (sanjaychitransh) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:104]: job check means --example of configuration file

Comment (hailin0) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:104]: example file does not check whether the code implementation is consistent and correct with the job configuration

Comment (hailin0) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:104]: You can reference these test cases

https://github.com/apache/seatunnel/blob/c0487a9dd2a8e7b2628f58a333ec28cb72652dc6/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-iotdb-e2e/src/test/java/org/apache/seatunnel/e2e/connector/iotdb/IoTDBIT.java#L107

Comment (hailin0) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:104]: Submitting the job to run can prove that the newly added code is successfully executed.

Comment (hailin0) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcPrestoIT.java:104]: @sanjaychitransh

**Pull request closed without merging**